junit5-testing/src/test/java/org/jsmart/zerocode/tests/loadjupiter/JUnit5LoadTest.java

```diff
@@ -0,0 +1,24 @@
+package org.jsmart.zerocode.tests.loadjupiter;
+
+import org.jsmart.zerocode.core.domain.LoadWith;
+import org.jsmart.zerocode.core.domain.TestMapping;
+import org.jsmart.zerocode.core.domain.TestMappings;
+import org.jsmart.zerocode.jupiter.extension.ParallelLoadExtension;
+import org.jsmart.zerocode.tests.jupiter.JUnit5Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith({ParallelLoadExtension.class})
+public class JUnit5LoadTest {
```

Comment:
More examples in HelloWorld repo will be helpful...

Reply:
will do.
Already a PR with few handy examples to unblock the user are there!
PR link: https://github.com/authorjapps/performance-tests/pull/3/files